tests/sample_sheet.csv

```diff
@@ -1,3 +1,2 @@
 name,reads,reads2
-SRR10903401,SRR10903401_R1.fastq,SRR10903401_R2.fastq
-SRR10903402,SRR10903402_R1.fastq,SRR10903402_R2.fastq
+Test,Test_R1.fastq,Test_R2.fastq
```

Comment:
Why change the names in the sample sheet?

Reply:
I changed this so that the how-to document makes sense. The 'SRR...' reads were just used by me locally. 'Test' is gonna be the small dummy sample included to the repository. Probably there will be more dummy samples in the near future for different time and geo stamps. The sample_sheet will then be adapted accordingly.